### ISSUE TITLE
Windows compatibility

### ISSUE THREAD

**Issue by db4** [janestreet/base#71]
Trying to compile `base` under windows I get:
```
File "_build/default/src/mpopcnt.sexp", line 1, characters 1-11:                                          
12:411 | '(-mpopcnt)'
     ^^^^^^^^^^
Error: too many s-expressions found in input
```
After the following
```diff
--- a/src/dune
+++ b/src/dune
@@ -16,7 +16,7 @@
 
 (library (name base) (public_name base)
  (libraries caml sexplib0 shadow_stdlib) (install_c_headers internalhash)
- (c_flags :standard -D_LARGEFILE64_SOURCE (:include mpopcnt.sexp))
+ (c_flags :standard -D_LARGEFILE64_SOURCE)
  (c_names exn_stubs int_math_stubs internalhash_stubs hash_stubs am_testing)
  (preprocess no_preprocessing)
   (lint 
```
`base` builds OK, but I expect `-mpopcnt` option should be excluded automatically if not supported by C compiler (MSVC in my case)?

**Comment by seliopou** [comment]
Thanks for reporting this.

This has been a perennial proble that I believe is in the process of being fixed internally. See #60 for some discussion. I'll leave this open until @diml's change lands.

**Comment by hhugo** [comment]
Fixed in base v0.12.1. see https://github.com/ocaml/opam-repository/pull/14171.
